displayio/_epaperdisplay.py

```diff
@@ -36,38 +41,38 @@ class EPaperDisplay:
 
     def __init__(
         self,
-        display_bus,
-        start_sequence,
-        stop_sequence,
+        display_bus: _DisplayBus,
+        start_sequence: _typing.ReadableBuffer,
+        stop_sequence: _typing.ReadableBuffer,
         *,
-        width,
-        height,
-        ram_width,
-        ram_height,
-        colstart=0,
-        rowstart=0,
-        rotation=0,
-        set_column_window_command=None,
-        set_row_window_command=None,
-        set_current_column_command=None,
-        set_current_row_command=None,
-        write_black_ram_command,
-        black_bits_inverted=False,
-        write_color_ram_command=None,
-        color_bits_inverted=False,
-        highlight_color=0x000000,
-        refresh_display_command,
-        refresh_time=40,
-        busy_pin=None,
-        busy_state=True,
-        seconds_per_frame=180,
-        always_toggle_chip_select=False,
-        grayscale=False,
+        width: int,
+        height: int,
+        ram_width: int,
+        ram_height: int,
+        colstart: int = 0,
+        rowstart: int = 0,
+        rotation: int = 0,
+        set_column_window_command: Optional[int] = None,
+        set_row_window_command: Optional[int] = None,
+        set_current_column_command: Optional[int] = None,
+        set_current_row_command: Optional[int] = None,
+        write_black_ram_command: int,
+        black_bits_inverted: bool = False,
+        write_color_ram_command: Optional[int] = None,
+        color_bits_inverted: bool = False,
+        highlight_color: int = 0x000000,
+        refresh_display_command: int,
+        refresh_time: float = 40,
```

Comment:
Should be `int` or possible the default should be `40.0` ?

Reply:
I copied off https://circuitpython.readthedocs.io/en/latest/shared-bindings/displayio/index.html#displayio.EPaperDisplay, but I'm fine with adding a .0 to the end.